spec/01-unit/01-db/01-schema/01-schema_spec.lua

```diff
@@ -3,21 +3,6 @@ local cjson  = require "cjson"
 local helpers = require "spec.helpers"
 
 
-local luacov_ok = pcall(require, "luacov")
```

Comment:
for whatever reason, this line causes [this OpenResty assertion](https://github.com/openresty/lua-nginx-module/blob/16de4bea620b6281f948b0d65341e31b2c90d597/src/ngx_http_lua_util.c#L1181-L1182) to fail when `luacov` is installed.

Reply:
if you run busted with the `--coverage` option, then busted will load luacov before starting the tests. No need to load it from the `helpers.lua` file.

Reply:
That makes sense @Tieske, this block should be no longer needed so this probably doesn't matter, but I was wondering what was causing that weird assertion failure.

Reply:
I'm surprised it works at all. Since OpenResty broke the coroutines, and LuaCov uses coroutines to do its tracing. And besides that there are the debug hooks, which don't play nice with LuaJIT.